module/documents/character.mjs

```diff
@@ -0,0 +1,28 @@
+import ActorEd from "./actor.mjs";
+
+export default class CharacterEd extends ActorEd {
+
+    async _preCreate( options, userId ) {
```

Comment:
I don't think we need an extra subclass for that one thing, since we have to have the typeguard anyways. We can use Polymorphisms, but then we should do it systematically, like here: https://foundryvtt.wiki/en/development/guides/polymorphism-actors-items